src/components/User.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+const data = require('../data/data.json')
+
+const users = data
+
+User.propTypes = {
+  userData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string,
+      department: PropTypes.string.isRequired,
+    })
+  ),
+  skills: PropTypes.arrayOf(
+    PropTypes.shape({
+      skill1: PropTypes.string.isRequired,
+      skill2: PropTypes.string,
+      skill3: PropTypes.string,
+      skill4: PropTypes.string,
+      skill5: PropTypes.string,
+      skill6: PropTypes.string,
+    })
+  ),
+}
+
+export default function User() {
+  return (
+    <Wrapper>
+      {users.map(({ id, name, phone, email, department, skills }, index) => (
+        <Details>
+          <li key={index}>{id}</li>
```

Comment:
No need to map over this here again. `User` represents _one_ user object and you already map over your data in the `UsersPage`. Also, the `key` always needs to be on the first element of your map, in this case that's `Details` and I also wouldn't use the index as key but the id.

Reply:
Semantically I wouldn't put the details like name, phone, email, etc. in a list. It doesn't seem right. Skills on the other hand should be a list.